### ISSUE TITLE
Take Fresca YAML as input?

### ISSUE THREAD

**Issue by SevereOverfl0w** [hacksalot/HackMyResume#105]
FRESCA supports YAML as input, would HackMyResume want to extend it's engine to support taking .yml files as input?

**Comment by hacksalot** [comment]
Yes. Starting in v1.7.0 HMR will accept "equivalent" YML and/or XML for any resume input -- including both FRESH/FRESCA and JSON Resume formats.

**Comment by hacksalot** [comment]
Also mentioned in #11.

**Comment by n3f** [comment]
Will this support multiple yaml files, e.g. `hackmyresume build resume1.yml resume2.yml TO resume.json` ? also is there a format assumed (jrs or fresh)?

**Comment by zachdecook** [comment]
I like the idea of yaml as authoritative input. Here's my shell one-liner to get that ability:

```sh
cat resume.yml |python3 -c 'import json,sys;from ruamel.yaml import YAML;yaml=YAML(); print(json.dumps(yaml.load(sys.stdin)))' |hackmyresume build /dev/stdin TO index2.html
```

Note: requires `pip3 install ruamel.yaml`
